src/app/views/dashboard/stickers/stickers.ts

```diff
@@ -35,6 +35,8 @@ export type StickerCount = {
 	sticker: Sticker;
 };
 
+const FETCH_STICKERS = '/web/stickers/dash';
```

Comment:
name it FetchStickersEndpoint. I try not to ALLCAPS in TS, but instead use TitleCase